basics/indexing/forward-index.md

```diff
@@ -1,15 +1,29 @@
 # Forward Index
 
-The values for every column are stored in a forward index, of which there are three types:
+At conceptual level, the forward index is a map from document id (aka row index) to the actual column value of the row.
+How this map is implemented depends on the index encoding and on whether the column is sorted or not.
```

Comment:
```suggestion
The forward index implementation depends on the index encoding and whether the column is sorted or not.
```

Reply:
I think it makes sense to use map here. We said that the forward index is a map and now we say that this map is implemented in a specific way depending on some variables.